src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
let's set `isLoading` by default `true` and only change when we get people from server